src/vs/workbench/contrib/files/browser/fileActions.ts

```diff
@@ -283,7 +283,7 @@ function getDeleteMessage(distinctElements: ExplorerItem[]): { message: string,
 		return { message: nls.localize('confirmDeleteMessageFolder', "Are you sure you want to permanently delete '{0}' and its contents?", distinctElements[0].name), detail: '' };
 	}
 
-	return { message: nls.localize('confirmDeleteMessageFile', "Are you sure you want to permanently delete '{0}'?", distinctElements[0].name), detail: '' };
+	return { message: nls.localize('confirmDeleteMessageFile', "Are you sure you want to delete '{0}'?", distinctElements[0].name), detail: '' };
```

Comment:
Is this a message where you should add the warning about not being able to restore after exit?

Reply:
Yep, that was the idea.